README.md

```diff
@@ -28,24 +28,25 @@ definition. All development is done in the following repositories:
 
 ## Releasing
 
-Since the ``ansys-grantami-serverapi-openapi`` package is auto-generated, the release process differs slightly from
-the standard [PyAnsys release procedure](https://dev.docs.pyansys.com/guidelines/dev_practices.html#release-procedures).
+Since the ``ansys-grantami-serverapi-openapi`` package is auto-generated from the Server API documentation, the 
+versioning scheme will follow the Ansys Granta MI versioning scheme: ``YEAR.RELEASE``, e.g. ``2023.2``.
+
+Release candidates are versioned as ``YEAR.RELEASE.PATCH.rc<ITERATION>``.
+
+The release process follows the standard [PyAnsys release guidelines](https://dev.docs.pyansys.com/how-to/releasing.html).
 
 1. Ensure the ``main`` branch build status is green, which indicates that the most recent run of the ``CI`` workflow was successful.
-2. Create a new branch from the ``main`` branch with the name ``release/MAJOR.MINOR`` (for example, release/0.2).
+2. Create a new branch from the ``main`` branch with the name ``release/MAJOR.MINOR`` (for example, release/2023.2).
 3. Make the following changes in ``ansys-grantami-serverapi-openapi/pyproject.toml``:
-    - Set the ``version`` to ``MAJOR.MINOR``.
+    - Set the ``version`` to ``MAJOR.MINOR.0rc0``.
     - Set the ``Development Status`` classifier to ``Development Status :: 5 - Production/Stable``.
-4. Commit this file. Push the branch to GitHub and create a new PR for this release that merges it to ``main``.
-   While effort is focused on the release, changes should not be made to either the YAML definition
-   or the ``openapi-client-template``.
-5. Wait for the PyAnsys developers to functionally test the new release. Testers should locally install this
-   branch and use it to run the full suite of tests in the ``main`` branch of the pythonic packages.
-6. Tag the release:
+4. Commit this file. Push the branch to GitHub. Tag the release:
    ```console
-   git tag v<MAJOR.MINOR.0>
+   git tag v<MAJOR.MINOR.0rc0>
    git push origin --tags
    ```
+5. From this point on, changes that would result require regenerating the client code will be made in the ``main`` branch, 
```

Comment:
```suggestion
5. From this point on, changes that would require regenerating the client code will be made in the ``main`` branch, 
```

Reply:
I'm just thinking if this is the right approach to take. Should we just say that any changes made on the release branch upstream will go directly into the release branch here, and any changes made on the main branch upstream will go into the main branch here.

That way, we make sure we handle the case of any weird "release branch only" fixes upstream. I can't think of what they would be, but it's conceivable.

The downside of this would be that the branches then become separate, and it then becomes a bad idea to cherry-pick _anything_. For example, a there's a post-RC JSON change that we put into the release branch only, and then we have to update the template. In this scenario, we'd have to bump the template version and regenerate the code separately.

I don't think this is a scenario we have to worry about though, I think anything that goes into the release branch upstream is always merged into the main branch. So this is probably fine as is.

Reply:
I wanted avoid detailing all scenarios, whether changes are on JSON for the current release or the next release, or whether changes are on code generation, etc.
Perhaps we could remove this line entirely.

Reply:
I think that's probably the right thing to do. Either we say something sufficiently vague like "From this point on, changes that would require regenerating the client code will be made on the appropriate branch given the specifics of the situation" - or we just say nothing at all and we take that decision as and when we need to.

So yes, I vote for removing point 5.